backend/dataall/aws/handlers/quicksight.py

```diff
@@ -227,6 +256,53 @@ def register_user(AwsAccountId, UserName, UserRole='READER'):
             )
         return Quicksight.describe_user(AwsAccountId, UserName)
 
+    @staticmethod
+    def register_user_in_group(AwsAccountId, UserName, GroupName, UserRole='READER'):
```

Comment:
Same as previous comment, can we replace calls to `register_user(AwsAccountId,UserName, UserRole='READER')` with your new function `register_user_in_group(AwsAccountId,UserName, 'dataall', UserRole='READER')` to avoid redundancy in the code ?

Reply:
Same, thank you